### ISSUE TITLE
Noise addition Variance

### ISSUE THREAD

**Issue by kwunhangwong** [pytorch/opacus#697]
As from Gaussian differential privacy, I saw the paper implemented with an extra "4" variance, which cannot be observed from your noise addition optimizer code extracted from DP-SGD ...... 

May I know where do you, or do you add a standard deviation of '2' into the code? Thanks. 

<img width="613" alt="image" src="https://github.com/user-attachments/assets/585f2251-3889-4401-bf0d-46c35c9fe310">

**Comment by kwunhangwong** [comment]
Will this mean you add less noise but the measurement for epsilon are the same? thanks

**Comment by HuanyuZhang** [comment]
Yes. In other words, eps = 5 "replace" DP is stronger than eps = 5 "add-or-remove" DP.

**Comment by kwunhangwong** [comment]
Thanks a lot for your helpful reply, I will analyze your provided paper and close the issue, really appreciate your help!

**Comment by kwunhangwong** [comment]
> Yes. In other words, eps = 5 "replace" DP is stronger than eps = 5 "add-or-remove" DP.

Sorry, i could not find any replacement in the uniform sampling scheme... Could you give a bit more details about replacement in [Gaussian differential privacy](https://arxiv.org/pdf/1905.02383)?

**Comment by HuanyuZhang** [comment]
It is not about the algorithm, but about the privacy definition. For the same algorithm under different privacy definitions, the required noise amount is different (definition 1.1 in https://arxiv.org/pdf/2103.00039)
